### PR TITLE
Add license step to k8s install instructions

### PR DESCRIPTION
**Pull request by jwilander**
Adding this quickly because I forgot to include it before.

### CODE REVIEW

File: source/install/install-kubernetes-mattermost.rst [hunk 1]
```diff
@@ -5,7 +5,29 @@ Deploy a Mattermost Installation
 
 Deploying a Mattermost installation is very simple and just requires creating and applying a single manifest.
 
-**1. Create your installation manifest file**
+**1. (Enterprise Only) Create your Mattermost license secret**
```
Comment (justinegeffen) [inline source/install/install-kubernetes-mattermost.rst]: Could this be an independent point (not numbered) to clarify that it's only applicable to Enterprise customers?

**Comment by justinegeffen** [comment]
Other than my suggestion above, this looks good to me.

File: source/install/install-kubernetes-mattermost.rst (hunk 1 above)
Comment (amyblais) [inline source/install/install-kubernetes-mattermost.rst]:
```suggestion
**1. (Enterprise only) Create your Mattermost license secret**
```

Comment (jwilander) [reply source/install/install-kubernetes-mattermost.rst]: Not exactly sure how that would work but feel free to make a PR afterwards to update it :) I'm just going to merge this as is to get something out there

**Pull request merged**